vars/suse-15.yml

```diff
@@ -14,5 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+_rpm_key:
```

Comment:
using Ansible's `zypper_repository` usually don't required for manually rpm key import, could be done automatically ;-)